Cython/Utility/Coroutine.c

```diff
@@ -504,8 +506,13 @@ static int __Pyx_PyGen__FetchStopIterationValue(PyThreadState *$local_tstate_cna
             value = Py_None;
         }
         else if (likely(__Pyx_IS_TYPE(ev, (PyTypeObject*)PyExc_StopIteration))) {
+            #if CYTHON_COMPILING_IN_LIMITED_API
+            value = PyObject_GetAttrString(ev, "value");
```

Comment:
I think we can afford wasting a prepared Python string for this in non-limited API targets. Same further down.